### DISCUSSION TITLE
My account is not visible to the public

### DISCUSSION THREAD

**Discussion by Muzhgan-Rustaqi** [community/community#128896] [Repositories]
### Select Topic Area

General

### Body

I have created my git account a few days ago, even though I submitted my project as public visibility it is not visible to the public.

**Comment by davevad93** [comment]
Hi @Muzhgan-Rustaqi ,

I see in your profile that you have a `public` repo named [Youtube_Data_Analysis](https://github.com/Muzhgan-Rustaqi/Youtube_Data_Analysis), is that the project you are talking about?
